### ISSUE TITLE
toString removes newlines between a list of elements

### ISSUE THREAD

**Issue by gylaz** [cheeriojs/cheerio#1387]
Consider the following code:
```js
    const $ = cheerio.load(`
      <ul>
        <li>One</li>
        <li>Two</li>
        <li>Three</li>
      </ul>
    `);
```
If we call `$('li').toString()`, the output will be all of the `li` elements on one line:

```
<li>One</li><li>Two</li><li>Three</li>
```

It does not preserve the newlines and makes it inconvenient for longer lines, and it gets hard to parse visually when debugging issues.

`html()` does preserve newlines, but requires one target by the parent element.
`$('ul').html()` yields:
```
    <ul>
       <li>One</li>
       <li>Two</li>
       <li>Three</li>
     </ul>
```
But sometimes targeting the parent isn't possible, as I may want to target specific elements within that parent (say only elements with specific class or data attribute).

The question, is removing of the newlines a bug or a feature of `toString`? Is there another way to achieve what I want (target a list of specific elements preserving newlines)?

**Comment by 5saviahv** [comment]
Removing newlines in this instance is not bug, it is way how DOM elements work in browsers. It is also how they work in jQuery and also in Cheerio.
